cspell.json

```diff
@@ -0,0 +1,2195 @@
+    "tmpdir",
+    "tmpfs",
+    "tmpname",
+    "tmux",
+    "tomdoherty",
+    "toolchain",
+    "toplevel",
+    "totalentries",
+    "traceback",
+    "traceroute",
+    "trainwreck",
+    "triaging",
+    "tuid",
+    "tvfs",
+    "txqueuelen",
+    "txvlan",
+    "typecode",
+    "tzdata",
+    "tzutil",
+    "ucanhaz",
+    "uchar",
+    "udiff",
+    "uids",
+    "uint",
+    "ulimit",
+    "ulong",
+    "umask",
+    "umount",
+    "uname",
+    "unattend",
+    "uncask",
+    "uncategorized",
+    "uncommented",
+    "uncompress",
+    "unencrypted",
+    "unescaping",
+    "unforked",
+    "unformatter",
+    "unhold",
+    "unignored",
+    "uninst",
+    "unintuitive",
+    "unixy",
+    "unmanaged",
+    "unmerge",
+    "unmergeables",
+    "unmount",
+    "unmounting",
+    "unredacted",
+    "unregister",
+    "unregistering",
+    "unscoped",
+    "unscrubbed",
+    "unsecure",
+    "unshift",
+    "untag",
+    "untaint",
+    "untampered",
+    "untap",
+    "untar",
+    "untracked",
+    "untrusted",
+    "unversioned",
+    "upcase",
+    "uploader",
+    "uploaders",
+    "upsert",
+    "upto",
+    "urllib",
+    "urlopen",
+    "usag",
+    "usags",
+    "useb",
+    "usebackq",
+    "usecount",
+    "usename",
+    "useradd",
+    "userdefaults",
+    "userdel",
+    "userinfo",
+    "usermod",
+    "ushort",
+    "usri",
+    "ustring",
+    "utime",
+    "uuid",
+    "uuidgen",
+    "uuidtools",
+    "vagrantup",
+    "valclient",
+    "varargs",
+    "variablevalue",
+    "vbox",
+    "vboxsf",
+    "vcenter",
+    "vendored",
+    "versio",
+    "versioncompare",
+    "versionlock",
+    "vfstab",
+    "vgroot",
+    "viewkind",
+    "vincentaubert",
+    "virt",
+    "virtualbox",
+    "visudo",
+    "vkhatri",
+    "vlan",
+    "vmware",
+    "voidcmd",
+    "vxfs",
+    "waitfor",
+    "waitpid",
+    "waitsem",
+    "wakeup",
+    "waym",
+    "wchars",
+    "webhook",
+    "webrick",
+    "webserver",
+    "webui",
+    "westus",
+    "whatavailable",
+    "whateverd",
+    "whatif",
+    "whatinstalled",
+    "whatprovides",
+    "whereis",
+    "whoami",
+    "whyrun",
+    "winbase",
+    "windef",
+    "windir",
+    "windmc",
+    "windres",
+    "winerror",
+    "winevt",
+    "winmgmts",
+    "winnt",
+    "winprog",
+    "winrm",
+    "wmic",
+    "woot",
+    "wordpress",
+    "workdir",
+    "workgroup",
+    "wrlinux",
+    "wstring",
+    "wtime",
+    "xabcz",
+    "xargs",
+    "xattr",
+    "xchar",
+    "xdigit",
+    "xenial",
+    "xinetd",
+    "xproto",
+    "yajl",
+    "yardoc",
+    "yieldparam",
+    "yocto",
+    "yolo",
+    "zanecodes",
+    "zenoss",
+    "zeproc",
+    "zmscwx",
+    "zoneinfo",
+    "zypp",
+    "zypper",
+    "Ásgeirsson"
+  ],
+  // flagWords - list of words to be always considered incorrect
+  // This is useful for offensive words and common spelling errors.
+  // For example "hte" should be "the"
+  "flagWords": [
+    "hte"
+  ],
+  "ignorePaths": [
```

Comment:
I tried to ignore most things that did not have any prose in them, things that would not ever be user facing, and directories with binaries in them.

I avoided all of `spec/**` because main tests are full of nonsense words and gibberish that would have taken a long time to go over.